contracts/Bridge/token/TokenController.sol

```diff
@@ -114,6 +114,7 @@ contract TokenController is Ownable, Pausable {
      */
     function setMintCap(address minter_, uint256 amount_) external {
         _ensureAllowed("setMintCap(address,uint256)");
+        require(amount_ > minterToMintedAmount[minter_], "New cap should be greater than minted tokens");
```

Comment:
I think we should throw an error instead of use a `require` statement

Reply:
[4ef1a4c](https://github.com/VenusProtocol/token-bridge/pull/8/commits/4ef1a4c27fd824d8d4915f44a0c6c37d714a07c0)